website/content/docs/platform/k8s/injector/index.mdx

```diff
@@ -50,15 +50,15 @@ useful to customize Vault Agent beyond what the provided annotations offer.
 ### Authenticating with Vault
 
 The primary method of authentication with Vault when using the Vault Agent Injector
-is the service account attached to the pod. At this time, no other authentication
-method is supported by the controller.
+is the service account attached to the pod. Other authentication methods can be configured 
+using annotations.
```

Comment:
Asked above, but can we include a list here on the auth methods that can be configured this way in here (or above)?

Reply:
It's all auto-auth methods. I'll just make a link to auto-auth doc.